### PR TITLE
Improve chart-testing docs

### PR DESCRIPTION
**Pull request by mmulholla**
1. updated "chartTesting" to "chart-testing"
2. Add info on using a config file.

### CODE REVIEW

File: docs/helm-chart-checks.md
Comment (isutton) [inline docs/helm-chart-checks.md:160]:
```suggestion
    $ chart-verifier verify --enable chart-testing --set-values config.yaml some-chart.tgz
```

**Review by isutton — CHANGES_REQUESTED** [review]
Only one copy missing, other than that LGTM.

File: docs/helm-chart-checks.md
Comment (baijum) [inline docs/helm-chart-checks.md:160]:
```suggestion
    $ chart-verifier verify --enable chart-testing --set-values config.yaml some-chart.tgz
```

**Review by isutton — APPROVED** [review]

**Review by baijum — APPROVED** [review]

**Comment by mmulholla** [comment]
Waiting for doc team reviews

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [inline docs/helm-chart-checks.md:134]: Suggestion: Replace this sentence with the following one as the 1st step:

1. You can configure the `chart-testing` check by performing one of the following steps:

- option 1 
- option 2

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [inline docs/helm-chart-checks.md:136]: Suggestion: Place this under the 1st step as the 1st option (as a bullet item).

Comment (Srivaralakshmi) [inline docs/helm-chart-checks.md:136]: Question: In this sentence: "Through the `--set` command line option, which in addition of configuring `chart-verifier` itself," I am not clear on:

-  what we mean by: "which in addition of configuring `chart-verifier` itself"?
- Is configuring `chart-verifier` a prerequisite to this step?
- What you mean when you say - "itself"?
- Is the `chart-verifier` automatic and done by the operator?
- What value add does it bring when you mention about the `chart-verifier` configuration in the step for `chart-testing` check configuration? Can we just avoid this? If yes, then I suggest:
"From the `--set` option in the Red Hat Helm chart-verifier CLI, set the configuration values of the `chart-testing` check as follows:"

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [inline docs/helm-chart-checks.md:148]: Suggestion: Place this under the 1st step as the 2nd option (as a bullet item).

Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md:148]: Suggestion: "Use a YAML file similar to the following example and from the the `--set-values` CLI option, configure the `chart-testing` check:

Example YAML output"

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [inline docs/helm-chart-checks.md:158]: Question: When you say "Once the file ..", Which file do you mean?

Suggestion: Put a note saying: "Wait for the file to persist". And then, add after the Note: "After the file is persisted, you can verify a chart for the 'chart-testing' check configuration as follows:

**Review by Srivaralakshmi — COMMENTED** [review]
@mmulholla Thanks for the opportunity. I have left some suggestions. PTAL.
Cc: @Preeticp

File: docs/helm-chart-checks.md
Comment (mmulholla) [reply docs/helm-chart-checks.md]: Done.

File: docs/helm-chart-checks.md
Comment (mmulholla) [reply docs/helm-chart-checks.md]: Done

Comment (mmulholla) [reply docs/helm-chart-checks.md]: I changed this to just be: 
* Option 1: Through the `--set` command line option: 

Nice a simple!

File: docs/helm-chart-checks.md
Comment (mmulholla) [reply docs/helm-chart-checks.md]: I finished up with:
* Option 2: Create a YAML file (config.yaml) similar to the following example:

and later:

Specify the file using the `--set-values` command line option:

Think it all looks a lot cleaner now.

File: docs/helm-chart-checks.md
Comment (mmulholla) [reply docs/helm-chart-checks.md]: See above.

**Review by dperaza4dustbit — APPROVED** [review]

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md]: When we say perform any one of the following steps, it implies that these are options. No need to call it out as Option 1 and 2.

Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md]: > * Option 1: Through the `--set` command line option:

This is an incomplete sentence and there is no instruction of the step for the user to perform.
Suggest to write similar to this: "From the --set option in the Red Hat Helm chart-verifier CLI, set the configuration values of the chart-testing check as follows:"

Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md]: Thanks!

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md]: > I finished up with:
> 
> * Option 2: Create a YAML file (config.yaml) similar to the following example:
> 

When we say perform any one of the following steps, it implies that these are options. No need to call it out as Option 1 and 2. Remove Option 2 and retain the bullet item and the sentence.

Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md]: > Specify the file using the `--set-values` command line option:

Question:
- Is this step common to both options 1 and 2?

Sugegstion:
If this step is only for option 2, and this is indeed a verification step for the chart-testing check configuration, then I suggest to have a lead line saying:

-  Perform the following steps:

1. Create a YAML file (config.yaml) similar to the following example:
Note: Wait for the file to persist. After the file is persisted, you can verify a chart for the 'chart-testing' check configuration.
2. To verify a chart, specify the `config.yaml` file using the `--set-values` option in the Red Hat Helm chart-verifier CLI:

File: docs/helm-chart-checks.md
Comment (Srivaralakshmi) [reply docs/helm-chart-checks.md]: @mmulholla Please see my comments above.

**Comment by Srivaralakshmi** [comment]
@mmulholla Nice work! I have left a few suggestions as review comments. Please incorporate them and intimate me. I would like to take a look at the final changes. Thanks!

**Pull request merged**
